etl/src/energuide/embedded/basement.py

```diff
@@ -267,7 +304,10 @@ def material(self) -> MaterialType:
 
     def to_dict(self) -> typing.Dict[str, typing.Any]:
         material = self._MATERIAL_TRANSLATIONS[self.material]
+        foundation = self._FOUNDATION_TRANSLATIONS[self.foundation_type]
```

Comment:
I think you could use `self._FOUNDATION_TRANSLATIONS.get(self.foundation_type)` here easily enough.